public/styles/main.css

```
@@ -0,0 +1,252 @@
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+#wrap h2 {
+  font-size: 1.4rem;
+  font-weight: bold;
+  margin-bottom: 32px;
+}
+
+#section-title {
+  font-family: 'Poppins', sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)